src/internal/converters/eml/eml.go

```diff
@@ -50,6 +51,92 @@ func formatAddress(entry models.EmailAddressable) string {
 	return fmt.Sprintf(addressFormat, name, email)
 }
 
+// getICalData converts the emails to an event so that ical generation
+// can generate from it.
+func getICalData(ctx context.Context, data models.Messageable) (string, error) {
+	msg := data.(*models.EventMessageRequest)
+	event := models.NewEvent()
```

Comment:
nit: could you please move this after the if-block below just to keep things a little cleaner? As far as I can tell it's not used if we hit the if-block anyway